tensorboard/components/tf_tensorboard/default-plugins.html

```diff
@@ -37,6 +38,7 @@
     'histograms': 'tf-histogram-dashboard',
     'projector': 'vz-projector-dashboard',
     'text': 'tf-text-dashboard',
+    'pr_curves': 'tf-pr-curve-dashboard',
```

Comment:
It might look better, æsthetically speaking on the tab label, if this was "pr-curves".

Reply:
Oh man ... I think this is linked with the plugin name in the backend, which is `pr_curves`. What do you think about in a separate change, we replace _ with a space in the navigation bar?

Reply:
Yep: right now we unfortunately have the restriction that the tab in the plugin name must equal the plugin name on the backend. That's probably not good to have long-term. If you look at [my proposed `dashboards.ts` from #202](https://github.com/tensorflow/tensorboard/pull/202/files#diff-fa4f39e09871545434c4f3b48dbdf3d5R35), you can see the approach I had in mind for resolving this. By giving each plugin a little configuration object, we can easily add an optional (or required) `tabName` field that takes the string `"PR Curves"` instead of `"pr_curves"`. I believe at the time we chose not to do this just for simplicity and ease of merging (as #205 was created around the same time and it wasn't worth reconciling), but we might want to do something like that now.

Reply:
I like @wchargin's idea of adding a tabName config property.